content/en/docs/components/model-registry/installation.md

```diff
@@ -26,6 +26,24 @@ Kubeflow Model Registry is available as an opt-in alpha component in Kubeflow Pl
 
 If you have deployed the Kubeflow manifests, you may follow [these instructions](https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream#readme) to deploy Model Registry; please raise any feedback on [`kubeflow/model-registry`](https://github.com/kubeflow/model-registry/issues).
```

Comment:
It's possible a user might do this before ever seeing the notes below. Maybe we need a note here that says see the section below if you are planning to use the Kubeflow UI? Not sure what scenario you would want to have MR in the kubeflow namespace if you plan to use the Kubeflow Platform / Dashboard

Reply:
You are right, I'll get a warning to follow next step if they plan to install MR UI